### PR TITLE
Added initial Resource Page to the Docs.

### PR DESCRIPTION
**Pull request by pme123**
Fixed also small bug in documentation (https://slinky.dev/docs/the-tag-api/ ) - please verify.

Added initial Resource Page to the Docs.

I added a link to the top- and the right navbar - it seems the right navbar was required that it worked.

image

### CODE REVIEW

File: docs/public/docs/the-tag-api.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ div(data-"columns" := "3")
 To add event listeners to elements, you can pass in an attribute pair assigning an event to a handler function. In Slinky, the event value is based on a type from the [Scala.js DOM](https://github.com/scala-js/scala-js-dom) library (this may change in the near future, see [PR #53](https://github.com/shadaj/slinky/pull/53)).
 
 ```scala
-input(onChange := (event) => {
-  println("the value of this input element was changed!")
-})
+input(onChange := (event => {
+        println(s"the value of this input element was changed!")
```
Comment (shadaj) [inline docs/public/docs/the-tag-api.md:51]: Don't think the string interpolation is necessary here

**Review by shadaj — CHANGES_REQUESTED** [review]
One small nit, but otherwise this looks awesome! Thanks so much for putting the time to pull together all these resources!

File: docs/public/docs/the-tag-api.md (hunk 1 above)
Comment (pme123) [reply docs/public/docs/the-tag-api.md]: done

**Review by shadaj — APPROVED** [review]
Looks great, thanks for the PR!

**Pull request merged**